api/envoy/config/common/matcher/v3/matcher.proto

```diff
@@ -16,6 +19,80 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 
 // [#protodoc-title: Unified Matcher API]
 
+message MatchTree {
+  message MatchAction {
+    oneof action {
+      bool skip = 1;
+
+      string callback = 2;
```

Comment:
What is this?

Reply:
An action that passes the provided string as callback to the filter. I'll go through and add some more comments